production.md

```diff
@@ -0,0 +1,108 @@
+
+If you need access to any of the following, please update [groups.yaml].
+
+### GKE instance
+
+publishing-bot is running in a GKE cluster named `aaa` in the `kubernetes-public`
+- [GKE project](https://console.cloud.google.com/kubernetes/list/overview?project=kubernetes-public)
+- [aaa cluster](https://console.cloud.google.com/kubernetes/clusters/details/us-central1/aaa/details?project=kubernetes-public)
+
+The cluster can be accessed by [k8s-infra-rbac-publishing-bot@kubernetes.io].
+To access the cluster, please see these [instructions].
+
+### What images does it use?
+
+Publishing-bot [images] can be pushed by [k8s-infra-staging-publishing-bot@kubernetes.io].
+
+### What commands are in this repo and how/when do i use them?
+
+Make sure you are at the root of the publishing-bot repo before running these commands.
+
+#### Populating repos
+
+This script needs to be run whenever a new staging repo is added in kubernetes/kubernetes
+
+```sh
+hack/fetch-all-latest-and-push.sh kubernetes
+```
+
+#### Deploying the bot
+
+```sh
+make validate build-image push-image deploy CONFIG=configs/kubernetes
+```
+
+### How to connect to the `aaa` cluster
+
+You can use the `Activate Cloud Shell` in the GCP console above and in that console, run the following command
+```
+gcloud container clusters get-credentials aaa --region us-central1 --project kubernetes-public
+```
+
+then run `kubectl` commands to ensure you can see what's running in the cluster.
+
+### What is running there?
+
+The `publishing-bot` runs in a separate kubernetes namespace by the same name in the `aaa` cluster.
+The manifests [here](https://github.com/kubernetes/publishing-bot/tree/master/artifacts/manifests) have the definitions
+for these kubernetes resources. Example below:
+
+````shell
+davanum@cloudshell:~ (kubernetes-public)$ kubectl get pv,pvc,replicaset,pod -n publishing-bot
+NAME                                                        CAPACITY   ACCESS MODES   RECLAIM POLICY   STATUS        CLAIM                                          STORAGECLASS   REASON   AGE
+persistentvolume/pvc-084a4d52-0a57-4f70-a76a-5d2d2667429d   100Gi      RWO            Delete           Bound         publishing-bot/publisher-gopath                ssd                     8h
+
+NAME                                     STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS   AGE
+persistentvolumeclaim/publisher-gopath   Bound    pvc-084a4d52-0a57-4f70-a76a-5d2d2667429d   100Gi      RWO            ssd            8h
+
+NAME                        DESIRED   CURRENT   READY   AGE
+replicaset.apps/publisher   1         1         1       45d
+
+NAME                  READY   STATUS    RESTARTS   AGE
+pod/publisher-cdvwj   1/1     Running   0          9h
+````
+
+### How do i see what the publishing bot is doing?
+
+you can stream the logs of the pod to see what the publishing-bot is doing
+```shell
+kubectl -n publishing-bot logs pod/publisher-cdvwj -f
+```
```

Comment:
worth to link to the bot issue in k/k?

Reply:
Added a new question+answer for the bot issue in k/k